thirdeye-ui/src/app/locale/languages/en-us.json

```diff
@@ -553,6 +554,7 @@
         "subscription-group-name-required": "Subscription Group name is required",
         "te-recipes": "ThirdEye Recipes",
         "the-setup-is-completed": "The setup is completed",
+        "there-have-been-no-entity-in-the-timePeriod": "There have been no {{entity}} in the {{timePeriod}}",
```

Comment:
`there-have-been-no-entity-in-the-timeperiod` or `there-have-been-no-entity-in-the-time-period`

Reply:
Implemented `there-have-been-no-entity-in-the-time-period`